### DISCUSSION TITLE
Add donation feature

### DISCUSSION THREAD

**Discussion by 0ttik** [isar/isar#430] [Ideas]
This database is so amazing (including perfect mutli-isolate handling, speed, embedded inspector, etc., etc.) that I'm thinking from time to time that I would be happy to make a small donation to express my support for this software. WDYT about something like https://www.buymeacoffee.com/?

**Comment by simc** [comment]
Thank you very much! To be clear: all my projects are completely free and I don't expect any compensation.

If you want to support me anyway, consider doing it here: https://github.com/sponsors/leisim/
